models/NeatlineExhibit.php

```diff
@@ -81,6 +105,11 @@ public function getNumberOfRecords()
      */
     public function getRecordUrl($action = 'show')
     {
+        if ($action = 'show') {
+          $params = array('action' => $action, 'slug' => $this->slug);
+          return public_url($params, 'neatline');
```

Comment:
Changed from using non-ideal method for admin url detection to instead generating public URLs for all exhibit search results, in keeping with other plugins' approach.